### DISCUSSION TITLE
Bibliography is not showing up

### DISCUSSION THREAD

**Discussion by margaridaparrado** [joaomlourenco/novathesis#504] [Q&A] [answered]
Hey everyone! 

I'm using the most recent version of the thesis template (version 7.1.27) and the bibliography is not showing up at the end of the document and neither on the contents. Also when I try to cite, the citation shows up with the keyword (like this  [Moss85] ). I also get this warning: LaTeX: Citation 'Moss85' on page 24 undefined. 

I'm using Latex on vscode with MIKTex on macOS. I'm not sure if I've done something wrong with the installation but on my old computer (also macOS) it worked fine and now it doesn't. 

Hope anyone can help me fix this issue. 

Thank you :)

**Comment by joaomlourenco** [comment, marked as answer]
How are you compiling the document?  Please try:
```bash
latexmk -pdf -shell-escape template.tex
```
and let me know if you still have problems.

**Comment by margaridaparrado** [comment]
Thank you for the fast reply. I've done that and changed to bibtex on 2_biblatex and it worked just fine! Thank you so much
